rds-update-rdsh-collection/azuredeploy.json

```diff
@@ -227,7 +227,7 @@
               "typeHandlerVersion": "2.11",
               "autoUpgradeMinorVersion": true,
               "settings": {
-                "ModulesUrl": "[concat(parameters('_artifactsLocation'),'/DSC/Configuration.zip', parameters('_artifactsLocationSasToken'))]",
+                "ModulesUrl": "[concat(parameters('_artifactsLocation'),'DSC/Configuration.zip', parameters('_artifactsLocationSasToken'))]",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#creating-uris for how to create the uris

Reply:
fixed